build.gradle

```diff
@@ -17,6 +17,7 @@ apply plugin: "com.android.application"
 dependencies {
     compile "com.crossbowffs.remotepreferences:remotepreferences:0.1"
     provided "de.robv.android.xposed:api:53"
+    compile 'com.google.code.gson:gson:2.6.2'
```

Comment:
I don't think a new dependency should be added for parsing 2 lines of JSON. Use the `org.json.JSONObject` class instead (it's included in the Android SDK)